pandas/tests/test_format.py

```diff
@@ -3181,6 +3181,16 @@ def test_to_string_float_na_spacing(self):
                     '4       NaN')
         self.assertEqual(result, expected)
 
+    def test_to_string_without_index(self):
+        #GH 11729 Test index=False option
+        s= Series([1, 2, 3, 4])
+        result = s.to_string(index=False)
+        expected = (u(' 1\n') +
+                    ' 2\n' +
```

Comment:
It seems more logical to me that the space is not there if no index is added.

But, `DataFrame.to_string` also does this:

```
In [13]: print df.to_string()
   a  b  c
0  1  b  1
1  1  b  2
2  2  b  3
3  2  a  4

In [14]: print df.to_string(index=False)
 a  b  c
 1  b  1
 1  b  2
 2  b  3
 2  a  4
```

Of course not necessarily a reason to do it here as well